config/locales/en.yml

```diff
@@ -292,6 +292,8 @@ en:
     gem_members:
       authors_header: Authors
       owners_header: Owners
+      pushed_by: Pushed by
+      yanked_by: Yanked by
```

Comment:
Can you please remove ones added [here](https://github.com/rubygems/rubygems.org/blob/9df7769b29baf4c65159ef7ae7824a9c1aeee789/config/locales/en.yml#L264).
Also, you will need to add/remove these keys from all locale files to make tests pass.